src/frontends/pytorch/src/op/min_max.cpp

```diff
@@ -36,19 +37,47 @@ OutputVector translate_max(const NodeContext& context) {
         align_eltwise_input_types(context, x, y, true);
         return {context.mark_node(std::make_shared<v1::Maximum>(x, y))};
     }
-    // torch.max(input, dim, keepdim), returns values and indicies
+    // torch.max(input, dim, keepdim), returns values and indices
     auto axes_node = context.get_input(1);
     auto axis_const = context.const_input<int64_t>(1);
     auto keepdims = context.const_input<bool>(2);
     auto values = context.mark_node(std::make_shared<v1::ReduceMax>(x, axes_node, keepdims));
     auto k = context.mark_node(std::make_shared<v0::Constant>(element::i32, Shape{}, 1));
     auto topk =
         context.mark_node(std::make_shared<v3::TopK>(x, k, axis_const, v3::TopK::Mode::MAX, v3::TopK::SortType::NONE));
-    auto indicies = context.mark_node(std::make_shared<v0::Convert>(topk->output(1), element::i64));
+    auto indices = context.mark_node(std::make_shared<v0::Convert>(topk->output(1), element::i64));
     if (!keepdims) {
-        indicies = context.mark_node(std::make_shared<v0::Squeeze>(indicies, axes_node));
+        indices = context.mark_node(std::make_shared<v0::Squeeze>(indices, axes_node));
     }
-    return {values, indicies};
+    return {values, indices};
+};
+
+OutputVector translate_max_dim(const NodeContext& context) {
+    // torch.max (same for torch.min) actually has two interfaces smashed together:
+    // torch.max(x, dim, keepdim) and torch.max(x, y)
```

Comment:
Why you have the same comment here as in `translate_max`?

Reply:
Changed the comment